lib/connections/service.go

```diff
@@ -276,7 +276,7 @@ func (s *service) handle(ctx context.Context) {
 		ct, connected := s.model.Connection(remoteID)
 
 		// Lower priority is better, just like nice etc.
-		if connected && ct.Priority() > c.priority {
+		if connected && (ct.Priority() > c.priority || time.Since(ct.Statistics().StartedAt) > 10*time.Second) {
```

Comment:
This would be a good place for a named constant, `minConnectionReplaceAge`?

Reply:
I think you are right. I added it.